Rules.md

```diff
@@ -1,16 +1,16 @@
 # Definitions
 
-* **Turn:** A single iteration of the game flow. There are a maximum of 10 turns in a game.
+* **Round:** A single iteration of the game flow. There are a maximum of 10 rounds in a game.
```

Comment:
For the same reason I refactored 'turn' to 'round' I find it strange to call it 'turns', as you often say 'taking a turn', meaning only a single player.

I couldn't find a better term for round in battles (except for "battle round" if you'd like), and as this is only mentioned twice in only a single sentence in the entire document, I don't think it really clashes.